### PR TITLE
docs: locators aka matchers

### PR DESCRIPTION
**Pull request by vobu**
- docs for bespoken
- prep "todo"-style tests of locator-/matcher-combinations yet untested

### CODE REVIEW

File: docs/locators.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+await control.press()
+```
+
+Here are couple of samples to get you comfortable with `wdi5`'s/OPA5's locators/matchers.
+
+?> Note: all of them are essentially taken from the `/examples/ui5-js-app/webapp/test/e2e/**/*.test.js` files.
+
+## ancestor
+
+Get a "child" control from a "parent" (ancestor).
```
Comment (Siolto) [inline docs/locators.md:34]: applies also for the descendent matcher: as far as I know the optional parameter `direct` as described in the [API](https://sapui5.hana.ondemand.com/#/api/sap.ui.test.matchers.Ancestor) is currently not possible. So the locator searches all parents/descendents in the hierarchy and not only the next one in the DOM Tree

File: docs/locators.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  const input = await browser.asControl(inputSelector)
+  await input.enterText(value)
+
+  expect(await input.getValue()).toEqual(value)
+})
+```
+
+## controlType
+
+Specify a UI5 control by namespace for retrieval.
```
Comment (Siolto) [inline docs/locators.md:81]: I think the description matches more the `viewName` locator and not the `controlType`^^

File: docs/locators.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+## i18n
+
+Retrieve a control by it's internationalization key rather than by its' visible text.
+
+```javascript
+it("check i18nText matcher user button", async () => {
+  const i18nSelector = {
+    selector: {
+      i18NText: {
+        propertyName: "text",
```
Comment (Siolto) [inline docs/locators.md:131]: the locator `i18nText` actually as more than those two parameters but I think we currently only support `propertyName` and `key` yet

File: docs/locators.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+## controlType
+
+Specify a UI5 control by namespace for retrieval.  
+Typically not used standalone, but in combination with any other locator.
+
+```javascript
+it("get a specific list item", async () => {
+  const selectorWithTitleProperty = {
+    selector: {w
```
Comment (Siolto) [inline docs/locators.md:87]:
```suggestion
    selector: {
```

File: docs/usage.md [hunk 5]
```diff
@@ -45,17 +45,23 @@ These are the supported selectors from [sap.ui.test.RecordReplay.ControlSelector
 
 |      selector | supported in `wdi5` |
 | ------------: | ------------------- |
-|          `id` | &check;             |
-|    `viewName` | &check;             |
-| `controlType` | &check;             |
+|   `anchestor` | &check;             |
```
Comment (Siolto) [inline docs/usage.md:48]:
```suggestion
|   `ancestor` | &check;             |
```

**Review by Siolto — CHANGES_REQUESTED** [review]
It looks like you are not finished yet, right? But here is what I found so far :)

File: docs/locators.md (hunk 3 above)
Comment (vobu) [reply docs/locators.md:131]: right on, prepped a dedicated test file

File: docs/locators.md (hunk 4 above)
Comment (vobu) [reply docs/locators.md:87]: uuups, removed :)

File: docs/usage.md (hunk 5 above)
Comment (vobu) [reply docs/usage.md:48]: eagle eye, thx

File: docs/locators.md (hunk 2 above)
Comment (vobu) [reply docs/locators.md:81]: re-worded this to make it more clear

File: docs/locators.md (hunk 1 above)
Comment (vobu) [reply docs/locators.md:34]: added the hint on both ancestor and descendant

**Comment by vobu** [comment]
> It looks like you are not finished yet, right? But here is what I found so far :)

i thought i was finished. you proved the contrary. thanks for the feedback! 👍 
putting this thing in NOW

**Pull request merged**